ember-headless-form/src/components/-private/field.hbs

```diff
@@ -0,0 +1,16 @@
+{{#let (unique-id) (fn @set @name) as |uuid setValue|}}
```

Comment:
Question: how come these `field` files are just `index` files inside of the field folder?

Reply:
Not sure what you mean here with `index` files?